bundles/binding/org.openhab.binding.plugwise/src/main/java/org/openhab/binding/plugwise/internal/PlugwiseBinding.java

```diff
@@ -106,7 +107,7 @@ public void updated(Dictionary<String, ?> config) throws ConfigurationException
 
     private Stick setupStick(Dictionary<String, ?> config) {
 
-        String port = (String) config.get("stick.port");
+        String port = ObjectUtils.toString(config.get("stick.port"), null);
```

Comment:
Ah, thanks for the explanation! While I also do not see a reason to keep it on Java 1.6, I agree that your change is the least invasive - so let's go for it!